plugins/out_cloudwatch_logs/cloudwatch_logs.c

```diff
@@ -344,6 +344,15 @@ static int cb_cloudwatch_init(struct flb_output_instance *ins,
     flb_output_upstream_set(upstream, ctx->ins);
     ctx->cw_client->host = ctx->endpoint;
 
+    struct mk_list *head;
+    struct flb_filter_instance *f_ins;
+    mk_list_foreach(head, &config->filters) {
+        f_ins = mk_list_entry(head, struct flb_filter_instance, _head);
+        if (strstr(f_ins->p->name, "kubernetes")) {
```

Comment:
If I'm reading this correctly, this auto-enables the `kubernete_metadata` feature in all `kubernetes` environments (rather than only auto-enabling in EKS environments)?
Or am I misinterpreting where we get `f_ins` from?

Reply:
This is to check if the filter kubernetes plugin is enabled for the log config.If it is enabled we know we have to send entity to PLE.